packages/module/src/Battery/Battery.tsx

```diff
@@ -80,7 +80,7 @@ export interface BatteryProps extends React.DetailedHTMLProps<React.HTMLAttribut
   className?: string;
 }
 
-const Battery: React.FunctionComponent<BatteryProps> = ({ severity, label, labelHidden, className, ...props }: BatteryProps) => {
+export const Battery: React.FunctionComponent<BatteryProps> = ({ severity, label, labelHidden, className, ...props }: BatteryProps) => {
```

Comment:
The batter props are not exported still

Reply:
@Hyperkid123 I think they are, but I may be missing something.